### PR TITLE
temporary fix due to PR398 + added TriggerInfoMerger module (with config tested) + clean up

### PR DESCRIPTION
**Pull request by gianipez**
added a temporary fix for allowing to run the trigger sequences w/o errors. Another PR will be needed to implement in the proper way the algorithm we currently use in the Online recon sequence.

### CODE REVIEW

**Comment by gianipez** [comment]
Hi folks, 

I just took advantage of this PR for committing a new module used to merge the different TriggerInfo objects crated on each trigger-path. The module is extremely simple. I also updated the python script in order to created Trigger config that include this merging step. I tested that it doesn't crash the code. I need more time to test reading this merged object with an analyzer module. So far, I just used the debug level to check that the number of objects was correct.

Cheers,

Giani

**Comment by gianipez** [comment]
@FNALbuild run build tests

**Comment by gianipez** [comment]
@FNALbuild run build tests

**Comment by gianipez** [comment]
I'm not sure what's going on with jenkins... I tested a clean build from mu2ebuild01 a couple of times and I see no problems. I cannot see the log file because my Fermilab VPN doesn't work.

**Comment by ryuwd** [comment]
Hi @gianipez,

A quick way to read jenkins "Log file" links is to copy the link from the github comment and paste it into mu2egpvm like this
```bash
curl -L <paste link> | less
```

e.g.
```
$ curl https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//518/artifact/scons.log | less
scons: Reading SConscript files ...
FileNotFoundError: [Errno 2] No such file or directory: 'gen/fcl/Trigger/OnSpillTrigMenu/unbiased/main_unbiasedTriggerInfoMerger.fcl':
  File "/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/Offline/SConstruct", line 130:
    env.SConscript(ss)
  File "/cvmfs/mu2e.opensciencegrid.org/artexternals/scons/v3_1_2/lib/python3.8/site-packages/SCons/Script/SConscript.py", line 597:
    return _SConscript(self.fs, *files, **subst_kw)
  File "/cvmfs/mu2e.opensciencegrid.org/artexternals/scons/v3_1_2/lib/python3.8/site-packages/SCons/Script/SConscript.py", line 286:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/Offline/Trigger/python/SConscript", line 17:
    sources,targets,command = genTriggerFcl.generate(m, False, False)
  File "/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/Offline/Trigger/python/genTriggerFcl.py", line 284:
    subEpilogInclude = appendEpilog(pathName, relProjectDir,
  File "/scratch/workspace/GitHubPRTests/mu2e-offline-build-test/Offline/Trigger/python/genTriggerFcl.py", line 110:
    subSubEpilogMergerFile     = open(subSubEpilogMergerFileName,"w")
  File "/cvmfs/mu2e.opensciencegrid.org/artexternals/python/v3_8_3b/Linux64bit+3.10-2.17/lib/python3.8/codecs.py", line 905:
    file = builtins.open(filename, mode, buffering)
```

**Comment by ryuwd** [comment]
(Apologies for spam, I was looking into why logfiles weren't being uploaded to GitHub anymore)

**Comment by gianipez** [comment]
Thank you so much @ryuwd !!!

**Comment by gianipez** [comment]
@FNALbuild run build tests

**Comment by brownd1978** [comment]
Can you please comment on the large number of deleted fcl files?  Are these obsolete, or will they be replaced eventually?

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Comment by brownd1978** [comment]
@FNALbuild run build test

File: Trigger/src/MergeTriggerInfo_module.cc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    art::EDProducer{config},
+    _debug   (config().debug())
+  {
+    produces<TriggerInfoCollection>    ();
+  }
+
+  void MergeTriggerInfo::produce(art::Event& event) {
+    std::unique_ptr<TriggerInfoCollection> tiCol(new TriggerInfoCollection);
+
+    // create the selector
```
Comment (bechenard) [inline Trigger/src/MergeTriggerInfo_module.cc:50]: I feel like I;m reading fortran code ith 72 columns.....

**Review by bechenard — APPROVED** [review]
I assume the fcl stuff is right and the rest of the code looks fine (even if the style made me cringe).

**Comment by bechenard** [comment]
I haven;t touched the fcl files, Giani did this so you should ask him

**Comment by gianipez** [comment]
Hi @brownd1978 ,

The FHICL files I deleted where used in the past for setting up the prescale filters used at the end of the trigger-paths to prescale w.r.t. the good events. A long time ago, I decided to remove them all, because there was no use case for it.

**Pull request merged**